src/pages/messages/components/NviStatusPage.tsx

```diff
@@ -0,0 +1,126 @@
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import {
+  Box,
+  IconButton,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { visuallyHidden } from '@mui/utils';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useFetchNviCandidates } from '../../../api/hooks/useFetchNviCandidates';
+import { useFetchOrganization } from '../../../api/hooks/useFetchOrganization';
+import { BackgroundDiv } from '../../../components/styled/Wrappers';
+import { RootState } from '../../../redux/store';
+import { OrganizationApprovalStatusDetail } from '../../../types/nvi.types';
+import { Organization } from '../../../types/organization.types';
+import { isValidUrl } from '../../../utils/general-helpers';
+import { getLanguageString } from '../../../utils/translation-helpers';
+
+interface NviStatusPageProps {
+  activeYear: number | string;
+}
+
+export const NviStatusPage = ({ activeYear }: NviStatusPageProps) => {
+  const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+
+  const organizationQuery = useFetchOrganization(user?.topOrgCristinId ?? '');
+  const institution = organizationQuery.data;
+
+  const nviQuery = useFetchNviCandidates({ size: 1, aggregation: 'all', year: activeYear });
+  const aggregationKeys = Object.keys(nviQuery.data?.aggregations?.organizationApprovalStatuses ?? {});
+  const aggregationKey = aggregationKeys.find((key) => isValidUrl(key));
+  const nviAggregations = nviQuery.data?.aggregations?.organizationApprovalStatuses[aggregationKey ?? ''] as
```

Comment:
Modellen for `organizationApprovalStatuses` er utrivelig å jobbe med, da vi må parse alle keys og slå opp på den som er URL nå, så det blir litt logikk i frontend. Har ikke fått gjennomslag for å oppdatere dette enda, så kan være det må bli sånn i første omgang :/